lib/adiwg/mdtranslator/readers/iso19115_2/modules/module_iso19115_2.rb

```diff
@@ -11,23 +11,21 @@ module Mdtranslator
       module Readers
          module Iso191152
             module Iso191152
-               def self.unpack(xMetadata, hResponseObj)
-                  intMetadataClass = InternalMetadata.new
-
-                  intObj = intMetadataClass.newBase
-                  @intObj = intObj
-                  @contacts = intObj[:contacts]
+               @intMetadataClass = InternalMetadata.new
```

Comment:
i see this is what you were referring to in your comment @rshewitt. why does this module specifically need to change and not the others, like https://github.com/GSA/mdTranslator/pull/29/files#diff-81d30c940445d9a7e8c66f815fb51af3f62d6c4718ce74ffade5b05e38166993R16?

Reply:
module_[schema].rb reader files, in this case `module_iso11915_2`, have helper functions to process internal contacts and setup things for testing which rely on things being instance vars. this isn't done elsewhere (e.g. `module_responsibility` ) because generally processing is contained within the `self.unpack` method so local/function scope vars are sufficient.

Reply:
it's reasonable to then question why i decided to use class vars for the xpaths in the modules and my only explanation is that i figured class vars would be rewritten less frequently and could made accessible to other modules for reusability. they're not ever rewritten nor are they really ever reused so it's kinda mute. they could be stored in the `self.unpack` method as local vars and things would be fine.

Reply:
thanks for the explanation!